spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -274,7 +279,22 @@ enum class SpdxOperator(
      */
     val priority: Int
 ) {
+    /**
+     * The conjunctive binary "AND" operator to construct a new license expression if required to simultaneously comply
+     * with two or more licenses, where both the left and right operands are valid [SpdxExpressions][SpdxExpression].
+     */
     AND(1),
+
+    /**
+     * The disjunctive binary "OR" operator to construct a new license expression if presented with a choice between
+     * two or more licenses, where both the left and right operands are valid [SpdxExpressions][SpdxExpression].
+     */
     OR(0),
+
+    /**
+     * Use the binary "WITH" operator to construct a new license expression if a set of license terms apply except under
+     * special circumstances. The left operand is a [SpdxLicenseIdExpression] value and the right operand is a
+     * [SpdxLicenseIdExpression] that represents the special exception terms.
```

Comment:
The right operand is a `SpdxLicenseExceptionExpression`.

Reply:
Done.